scripts/common/tasks.sh

```diff
@@ -145,7 +148,15 @@ function weave_reset() {
     DATAPATH=datapath
     CONTAINER_IFNAME=ethwe
 
-    WEAVE_TAG=${WEAVE_VERSION}
+    if [ -z "$WEAVE_TAG" ]; then
+        WEAVE_TAG=$(docker image ls | grep weaveworks/weave-npc | awk '{ print $2 }')
```

Comment:
```suggestion
        WEAVE_TAG=$(docker image ls | grep weaveworks/weave-npc | awk '{ print $2 }' | head -1)
```
In case it's an upgrade